### ISSUE TITLE
Split tooltips do not work on 'flags' series

### ISSUE THREAD

**Issue by simdevmon** [highcharts/highcharts#7006]
#### Expected behaviour
When I enable shared / split tooltips I would expect that tooltips on flag series are still working.

#### Actual behaviour
No tooltip is displayed anymore on flags series after shared tooltips were enabled.

#### Live demo with steps to reproduce
http://jsfiddle.net/simdevmon/hk50v1m3/1/

Without shared tooltips the tooltip of flags series are displayed correctly:
![2017-07-25_14h21_40](https://user-images.githubusercontent.com/16277781/28571861-0a519b1e-7145-11e7-9f63-b5b9b0b35ded.png)

After setting `shared: true` on tooltip, there is no tooltip displayed on flags series anymore:
![2017-07-25_14h21_57](https://user-images.githubusercontent.com/16277781/28571863-0b3c337c-7145-11e7-9777-d96452b46aa8.png)

#### Affected browser(s)
Chrome 59

**Comment by KacperMadej** [comment]
Hi,

Thank you for reporting about the bug. It's already resolved on master branch and the fix awaits next version release that could be expected in next week or two.

Duplicate of #6764

**Comment by simdevmon** [comment]
I updated to HighStock 6.0.1, but the problem still exists.

**Comment by pawelfus** [comment]
Shared works fine, take a look: http://jsfiddle.net/hk50v1m3/2/ - it looks like `split` is broken with single points here: https://github.com/highcharts/highcharts/blob/b39922516cdad8a880a7b8dbe13c23b4427deeb8/js/parts/Tooltip.js#L616-L617

Where we assume that `points` is always an array, not a single point.

**Workaround:**
```
(function(H) {
  H.wrap(H.Tooltip.prototype, 'renderSplit', function(proceed, text, pts) {
    return proceed.call(this, text, H.splat(pts));
  })
})(Highcharts);
```

Demo: http://jsfiddle.net/mofs0qs6/2/

**Comment by simdevmon** [comment]
To me it is not clear where `points` refer to when I look at the example. Is this a bug or is something wrong with my input data?

**Comment by pawelfus** [comment]
Sorry, I forgot to add labels, done - it's a bug :)
The paragraph with points is a note for ourselves, to speed up fixing this bug.

At this moment you have two workarounds:

- don't use `split`, use `shared` only
- use `split` with my snippet above

**Comment by TorsteinHonsi** [comment]
Fixed in v6.0.2.
